build.gradle

```diff
@@ -50,11 +50,21 @@ if (hasProperty('buildScan')) {
     buildScan {
         termsOfServiceUrl = 'https://gradle.com/terms-of-service'
         termsOfServiceAgree = 'yes'
-        def ns = System.getenv("NAMESPACEINUSE") ?: "invalid-namespace"
+        def project = ""; def cluster = ""; def nschunk = "";
+        if (System.getenv("NAMESPACEINUSE")) {
+            project = "broad-jade-integration"
+            cluster = "integration-master"
+            def ns = System.getenv("NAMESPACEINUSE")
+            nschunk = "%0Aresource.labels.namespace_name%3D%22${ns}%22%0Alabels.k8s-pod%2Fcomponent%3D%22${ns}-jade-datarepo-api%22"
+        } else {
+            project = System.getenv("GOOGLE_CLOUD_PROJECT")
```

Comment:
What environments will this work in now? It doesn't look like it would work in my dev account, although that is not super important.